.github/workflows/test.yaml

```diff
@@ -1,13 +0,0 @@
-name: Test & Build
-
-on:
-  push:
-
-jobs:
-  package:
-    uses: ./.github/workflows/build.yaml
-    with:
-      release: true
-    secrets:
-      DS_RELEASE_BOT_ID: ${{ secrets.DS_RELEASE_BOT_ID }}
-      DS_RELEASE_BOT_PRIVATE_KEY: ${{ secrets.DS_RELEASE_BOT_PRIVATE_KEY }}
```

Comment:
I think we will still need this to build and release the package.

We should most likely filter by branch name for that though.

Reply:
I'm a bit confused on this point.  Shouldn't this be fully handled by the build -> release workflows and only triggered by release tags?  I was unclear due to the use of nested workflows.